PacketDotNet/PacketDotNet.csproj

```diff
@@ -21,5 +21,6 @@
   </PropertyGroup>  
   <ItemGroup>
     <PackageReference Include="log4net" Version="2.0.8" />
+    <PackageReference Include="NUnitTestAdapter" Version="2.1.1" />
```

Comment:
Needs to be removed, we'd not want to pull this in for everyone.

Reply:
I can add Condition="'$(Configuration)' == 'Debug'" so it's not needed for release. It's needed to debug the test in Visual Studio.

Reply:
That would be better, but why you need this in the first place? The tests runs fine without it 😄

Reply:
What IDE do you use? I'm using VS2017 Enterprise and I couldn't get them to run in it without it.

Reply:
Also VS2017 (latest version). Tests run just fine here through VS itself (right-click project -> run unit tests).

Reply:
I think it's because I'm using the Enterprise version. I remember some post about needing this for it and possibly the professional version too.

I could kick off the test without this, but they wouldn't work in VS test Explorer.

Reply:
I use Community version but I used VS2017 Enterprise in the past and the tests worked fine as well. I do use quite some extensions so perhaps one of them does anything 👍

Reply:
Ah, I had the wrong Adapter (3) installed, it works with 2. Removing..